model/host/host.go

```diff
@@ -739,3 +744,53 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	}
 	return counts, nil
 }
+
+// find all containers
+func (h *Host) FindAllContainers() ([]Host, error) {
```

Comment:
The convention we use (and I'm like 50% torn about whether it's a good convention) is to just have a function that returns a query and have the caller do the Find part. So I would omit everything below line 753 and have the function return a db.Q object (same with the other ones)

Reply:
I think in this case, because the host struct is no longer the receiver of this function, that using the Find function here is okay.